### DISCUSSION TITLE
월드 심사에 올린 테스트 링크를 통해 여러명이 접속을 하면 각자 다른 공간에 접속하게 됩니다.

### DISCUSSION THREAD

**Discussion by icjoon** [naverz/zepeto-studio-kor#567] [5. Q&A]
월드 심사에 올린 테스트 링크를 통해 여러명이 접속을 하면 각자 다른 공간에 접속하게 됩니다.
유니티 에디터를 이용하여 로컬서버로 접속했을때는 정상작동하고 여러명이 다 같은 공간에서 만날 수 있습니다.
무슨 차이가 있어서 그렇게 되는지 궁금하여 질문 올립니다.

-제페토패키지 버전 1.1.5

-제페토 앱 버전 3.10.4

-Window 10에서 개발합니다

**Comment by Joyounwoo** [comment]
테스트 링크에서 여럿이 접속하는 기능은 아직 제공하지 않는 것으로 보입니다. 이는 저나 다른 개발자 분들도 마찬가지 상태입니다.
테스트 링크 빌드에서 월드 내 초대링크 등을 이용한 방법으로도 초대가 불가능하니, 로컬 환경에서만 테스트 해보셔야 할 것 같습니다.

**Comment by ttoowa** [comment]
퍼블리시 환경에서만 발생하는 버그들도 존재해 (QR코드로 접속시에도 발생하지 않는) 이를 확인하기 위해선 퍼블리시 환경에서 여럿이 테스트가 가능해야 할 것으로 보입니다.

저도 이 문제가 빨리 해결되면 좋겠네요

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 심사 중인 월드는 실제 제페토 서버에서 만나는 구조입니다.
개선 사항이 반영되기까지 시간이 걸릴 것으로 예상되며
현재는 아래와 같은 방법으로 초대가 가능합니다.

- 테스트할 계정은 모두 테스터로 등록이 되어야 하고, 모두 제페토 친구를 맺어야 합니다.
- 제페토 앱 버전 (3.11.0)으로 업데이트가 되어야 합니다.
- 방장이 방을 만들어 입장 후 친구 초대하기로 초대합니다. (초대 링크가 아니라 직접 초대를 합니다) 
- 방장외 테스터는 제페토 다른 월드가 아닌 로비에 있다가 초대를 수락해야 합니다.

방장은 가급적 `안드로이드` 환경에서 초대를 하기를 권장 드립니다.
초대를 받는 테스터는 기기에 상관이 없습니다.

**Reply by Chordcode002** [reply]
이 문제 현재는 해결되었나요? 관련 정보를 찾아볼 수가 없어서 해결이 된건지 아직 미해결인지 알 수가 없네요

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 심사 중인 월드에서 다 같이 만나는 방법은 초대를 통해야 한다는 부분은 현재까지도 동일합니다.
월드 버전 1.3.0 이후에 iOS에서 초대가 안되는 문제는 수정이 되었습니다. https://github.com/naverz/zepeto-studio-kor/discussions/677
심사 중인 월드에서 테스터들이 모두 같은 공간에서 만나도록 하는 개선 작업의 우선 순위가 높아질 수 있도록 조율해 보겠습니다.
감사합니다.
